.github/workflows/main.yaml

```diff
@@ -58,7 +58,16 @@ jobs:
         working-directory: ./src/github.com/${{ github.repository }}
 
   image-prepare:
-    runs-on: ubuntu-22.04
+    runs-on: ${{ matrix.runner }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - runner: ubuntu-22.04
+            arch: amd64
+          - runner: ARM64
+            arch: arm64
+
```

Comment:
Thanks for your contribution. I have an idea. Can we separate arm's CI test into a new file, such as `main-arm.yaml`? This will not affect the original CI test and can also be used to track task details. :)

Reply:
+1, main-arm64.yaml should be also fine

Reply:
Great idea!
I've extracted the Docker building images to main-arm64.yaml!
Once this works we could start enabling more jobs for ARM64!